sources/Content_Quality/missing_rambam_intros.py

```diff
@@ -0,0 +1,50 @@
+import django
+
+django.setup()
+
+from sefaria.model import *
+from sefaria.helper.schema import convert_simple_index_to_complex, convert_jagged_array_to_schema_with_default, \
+    insert_first_child
+
+
+def convert_children_to_schema_nodes(tref):
```

Comment:
it seems you forgot to use this

Reply:
I'll take a look again...

Reply:
Thanks @YishaiGlasner - was a mistake on my end, working off an example from last summer, and missed this in my clean up. Appreciate the extra eyes.